### PR TITLE
[plugin: commercejs] Add commercejs plugin

### PR DESCRIPTION
**Pull request by jaepass**
## Description

- Adding the Commerce.js plugin for testing

Thank you Aziz for the help!

### CODE REVIEW

**Comment by teleaziz** [comment]
Converted this to a draft PR, as it's not working as expected in it's current state, something about the `@chec/commerce` not playing right with the rollup configurations, will look into it next

**Comment by teleaziz** [comment]
The issues with this PR are fixed here https://github.com/jaepass/builder/pull/1 , once merged will release

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@bazel/ibazel": "^0.15.10",
     "@bazel/typescript": "^3.4.2",
     "@builder.io/block-publish": "^1.1.2",
+    "@types/chec__commerce.js": "^2.7.1",
```
Comment (teleaziz) [inline package.json:9]: This is the root package.json, please remove from this PR

File: package-lock.json [hunk 2]
```diff
@@ -2,6 +2,14 @@
   "requires": true,
   "lockfileVersion": 1,
```
Comment (teleaziz) [inline package-lock.json:3]: Same for this file, it's the root one, please remove from this PR

**Review by teleaziz — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (jaepass) [reply package-lock.json:3]: Which file do you mean here?

**Review by teleaziz — APPROVED** [review]
👍 👍 🎉

**Pull request merged**